src/node.cc

```diff
@@ -712,9 +713,9 @@ void Exit(const FunctionCallbackInfo<Value>& args) {
 }
 
 static Maybe<bool> ProcessEmitWarningGeneric(Environment* env,
-                                             const char* warning,
-                                             const char* type = nullptr,
-                                             const char* code = nullptr) {
+                                      const char* warning,
+                                      const char* type = nullptr,
+                                      const char* code = nullptr) {
```

Comment:
This is an odd whitespace change?

Reply:
Okay, disregard that, I think this got addressed in a later commit in this PR :+1: